### DISCUSSION TITLE
用 AI「復活」已故親人，微軟最新專利獲批准

### DISCUSSION THREAD

**Discussion by rumy127** [1101-GS4719-C/Python-Community-GS4719-C#1599] [General]
在AI越來越發達的情況下，已經可以成功模擬人類的外貌、聲音、甚至個性；但在道德倫理上，這件事仍然有相當兩極的評價，能夠利用AI讓身邊已故的親友「復活」，並能與其互動和對話，對於許多尚未走出傷痛的人而言固然是最大的安慰；但另一派人認為，這件事違反了自然的生命循環。我的看法是，當AI已經能夠徹底模擬人類的情感，就如同複製人，那自我的存在再也不是獨一無二的，那生命又與物品有何差別？不曉得大家對於用AI模擬親人的法如何？
https://koin.kcg.gov.tw/?p=10271

**Comment by lazywulf** [comment]
如果將線上的人們視為網路封包，似乎利用AI「復活」已故知人似乎不是太大的問題。我們反而要關注其他面向，譬如若有人利用這點來進行詐騙或其他犯罪行為等，而非完全禁止這項科技的發展。

**Reply by thz0000** [reply]
我覺得這跟嗎啡的應用很像，在醫療上適當的使用可救人於痛苦；反之，當人們開始濫用甚至從事不法交易，那就成為糟糕的工具了。
所以，我覺得應該應該要設立一條'線'，一條不踰矩道德和法律的線，並適當地運用AI技術造福人類，而非欺騙與毀滅。

**Comment by v0ouo** [comment]
之前我大一時就有人來中央大學演講過，他當時也是運用了AI科技把鄧麗君復活到屏幕上，AI復活的鄧麗君不只能跟校長溝通，就連唱自己的歌也能行雲流水，簡直就跟原本的一樣。不過我認為用AI模擬人還是一個很大的問題，畢竟現在還沒有一套完整的法律制度來明訂AI的相關事務，也許有些人會拿這些新科技來做壞事。因此我認為現在還是靜靜等待發展就好。

**Comment by davidteng00000** [comment]
這個功能固然創新且讓人眼前一亮，但是我認為能否幫助到死者的家屬實在是有待商榷，反倒是可能勾起家屬們難過的回憶，值得深思

**Comment by eric202501** [comment]
我認為此技術其實存在很巨大的隱患，鑒於現在並沒有明確的法規規範ai生成的產物，其可能增長假訊息傳遞，畢竟真的能以假亂真，可以用於詐騙等犯罪，甚至上綱至國家間的情報戰或資訊戰，造成混亂動盪

**Comment by Yanxxx7979** [comment]
我認為雖然AI能夠滿足在世的人，讓他們看見自己逝去的親人，但「復活」的那個人，仍然不是自己真正日思夜想的「人」，而只是一個容貌及軀殼，這樣真的是一件會讓周遭人幸福的事嗎?
